modules/gitops-argo-cd-round-robin-sharding-algorithm.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argo-cd-round-robin-sharding-algorithm_{context}"]
+= Enabling the round-robin sharding algorithm
+
+:FeatureName: The `round-robin` sharding algorithm
+include::snippets/technology-preview.adoc[]
+
+By default the Argo CD Application Controller uses the non-uniform `legacy` hash-based sharding algorithm to assign clusters to shards. This can result in uneven cluster distribution. You can enable the `round-robin` sharding algorithm to achieve more equal cluster distribution across all shards. 
+
+Using the `round-robin` sharding algorithm in the {gitops-title} provides the following benefits:
+
+* Ensure more balanced workload distribution
+* Prevent shards from being overloaded or underutilized
+* Optimize the efficiency of computing resources
+* Reduce the risk of bottlenecks
+* Improve overall performance and reliability of the Argo CD system
+
+The introduction of alternative sharding algorithms allows for further customization based on specific use cases. You can select the algorithm that best aligns with your deployment needs, which results in greater flexibility and adaptability in diverse operational scenarios.
+
+.Prerequisites
+* You have installed the {gitops-title} Operator in your cluster.
+* You have access to the {OCP} web console.
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+You can enable the `round-robin` sharding algorithm by either using the {OCP} web console or the command-line interface.
```

Comment:
Yes, thank you! The reason I included it as such was because I did not know how else to do it to make sense with having only one procedure. Having 2 separate procedures for this solves the issue :+1: